AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md

```diff
@@ -53,14 +53,34 @@ You will need to download and install the following toolkits, tools, and compone
 
 **1. Get Intel® AI Tools**
 
-Required AI Tools: <Tensorflow* ><!-- List specific AI Tools that needs to be installed before running this sample --> 
+Required AI Tools: <Intel® Extension for TensorFlow* (CPU)）><!-- List specific AI Tools that needs to be installed before running this sample --> 
 <br>If you have not already, select and install these Tools via [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html). AI and Analytics samples are validated on AI Tools Offline Installer. It is recommended to select Offline Installer option in AI Tools Selector.<br>
-or simple pip install in your current ready python environment 
+please see the [supported versions](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html).
+
+>**Note**: If Docker option is chosen in AI Tools Selector, refer to [Working with Preset Containers](https://github.com/intel/ai-containers/tree/main/preset) to learn how to run the docker and samples.
+
+**2. (Offline Installer) Activate the AI Tools bundle base environment**
+
+If the default path is used during the installation of AI Tools:
+```
+source $HOME/intel/oneapi/intelpython/bin/activate
 ```
-pip install tensorflow==2.14
+If a non-default path is used:
 ```
-please see the [supported versions](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html).
+source <custom_path>/bin/activate
+```
+ 
+**3. (Offline Installer) Activate relevant Conda environment**
 
+```
+conda activate tensorflow ## For the system with Intel CPU   
+``` 
+
+**4. Clone the GitHub repository**
+``` 
+git clone https://github.com/oneapi-src/oneAPI-samples.git
+cd oneAPI-samples/AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted
+```
 ## Run the Sample
 
 >**Note**: Before running the sample, make sure Environment Setup is completed.
```

Comment:
>**Note**: Before running the sample, make sure Environment Setup is completed.

Go to the section which corresponds to the installation method chosen in [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html) to see relevant instructions:
* [AI Tools Offline Installer (Validated)/Conda/PIP](#ai-tools-offline-installer-validatedcondapip)
* [Docker](#docker)

### AI Tools Offline Installer (Validated)/Conda/PIP

Run the Python script:
```
python TensorFlow_HelloWorld.py
```
### Docker
AI Tools Docker images already have Get Started samples pre-installed. Refer to [Working with Preset Containers](https://github.com/intel/ai-containers/tree/main/preset) to learn how to run the docker and samples.

## Example Output

Reply:
It seems like for .py sample the only step users needs to do after Environment Setup is to run a script. It will be the same for Offline, Conda and PIP. And then there is Docker option for which we point user to a different instruction on containers repo.
Above you can see what I think should be included after Run the sample heading and up to Expected Output. Other steps seems to be duplicating with Environment Setup and I suggest to remove them

Reply:
remove the duplicated part. please review thanks